packages/node/src/integrations/tracing/hapi/types.ts

```diff
@@ -250,26 +240,15 @@ export interface RequestEvent {
   error: object;
 }
 
-type RequestEventHandler = (request: Request, event: RequestEvent, tags: { [key: string]: true }) => void;
 interface ServerEvents {
-  on(criteria: 'request' | ServerEventCriteria<'request'>, listener: RequestEventHandler): void;
+  on(criteria: any, listener: any): void;
 }
 
-type RouteRequestExtType =
-  | 'onPreAuth'
-  | 'onCredentials'
-  | 'onPostAuth'
-  | 'onPreHandler'
-  | 'onPostHandler'
-  | 'onPreResponse';
-
-type ServerRequestExtType = RouteRequestExtType | 'onRequest';
-
 export type Server = Record<string, any> & {
   events: ServerEvents;
-  ext(event: ServerRequestExtType, method: Lifecycle.Method, options?: Record<string, any>): void;
+  ext(event: any, method: Lifecycle.Method, options?: Record<string, any>): void;
   initialize(): Promise<void>;
-  register(plugins: Plugin<any> | Array<Plugin<any>>, options?: Record<string, any>): Promise<void>;
+  register(plugins: Plugin<any>, options?: Record<string, any>): Promise<void>;
```

Comment:
hmm, is this not tightening the types? used to accept a plugin or an array of plugins, not we only accept a single plugin?

Reply:
I just saw we cannot do this anyway as hapi has 5 different method overwrites and no union type. I am going to change this to `any` as well!